src/applications/static-pages/scroll-top-button.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * This button is used for navigating the user back to the top of the page.
+ * It's to be used on long article pages.
+ * There is some complexity in the animation toggling function "scrollListener"
+ * html is located in src/site/components/up_to_top_button.html
+ */
+
+export default function initScrollToTopButton() {
```

Comment:
Instead of having the functions nested in `initScrollToTopButton`, can we flatten the structure by moving the nested functions outside of that, into the upper scope?

Reply:
Yeah we can flatten the js expose function structure by doing that. I can get to it in a little bit.

Reply:
@ncksllvn Ready to go!